src/zmalloc.c

```diff
@@ -652,7 +712,10 @@ size_t zmalloc_get_frag_smallbins(void) {
         assert(!je_mallctl(buf, &reg_size, &sz, NULL, 0));
 
         /* Number of used regions in the bin */
-        snprintf(buf, sizeof(buf), "stats.arenas." STRINGIFY(MALLCTL_ARENAS_ALL) ".bins.%d.curregs", j);
+        if (arena == UINT_MAX)
+            snprintf(buf, sizeof(buf), "stats.arenas." STRINGIFY(MALLCTL_ARENAS_ALL) ".bins.%d.curregs", j);
+        else
+            snprintf(buf, sizeof(buf), "stats.arenas.%u.bins.%d.curregs", arena, j);
```

Comment:
Why do we need them?
```suggestion
        snprintf(buf, sizeof(buf), "stats.arenas.%u.bins.%d.curregs", arena, j);
```

Reply:
and dont forget to remove 
```
#define STRINGIFY_(x) #x
#define STRINGIFY(x) STRINGIFY_(x)
```

Reply:
right, we don't need it anymore, thanks